src/app/games/hive/tests/HiveRules.spec.ts

```diff
@@ -905,6 +905,22 @@ describe('HiveRules', () => {
         // Then there should be none
         expect(dropLocations.size()).toBe(0);
     });
+    it('should compute the expected spider moves for a specific board', () => {
+        // Given a specific state with 3 possible spider moves
+        const board: Table<HivePiece[]> = [
+            [[], [S], [s], [b]],
+            [[], [], [], [B]],
+            [[b], [], [], [a]],
+            [[A], [], [G], []],
+            [[Q], [g], [], []],
+        ];
+        const state: HiveState = HiveState.fromRepresentation(board, 6);
+
+        // When computing the possible moves for the spider
+        const moves: MGPSet<HiveMoveCoordToCoord> = rules.getPossibleMovesFrom(state, new Coord(1, 0));
+        // Then we should have exactly 4 moves
+        expect(moves.size()).toBe(4);
+    });
```

Comment:
Given 3 there should be 4 :o 
SATANERIE !

Reply:
oups!